applications/GeoMechanicsApplication/tests/line_interface_elements/Neumann_multi_stage_reset_displacements/ProjectParameters_stage1.json

```diff
@@ -0,0 +1,152 @@
+{
+    "problem_data": {
+        "problem_name":         "Neumann_multi_stage",
+        "start_time":           0.0,
+        "end_time":             2.0,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "U_Pw",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "3+3_line"
+        },
+        "material_import_settings":              {
+            "materials_filename":       "../common/MaterialParameters.json"
+        },
+        "time_stepping":              {
+            "time_step":            1.0
+        },
+        "buffer_size":                        2,
+        "echo_level":                         1,
+        "clear_storage":                      false,
+        "compute_reactions":                  false,
+        "move_mesh_flag":                     false,
+        "reform_dofs_at_each_step":           false,
+        "nodal_smoothing":                    false,
+        "block_builder":                      false,
+        "solution_type":                      "Quasi-Static",
+        "scheme_type":                        "Newmark",
+        "reset_displacements":                false,
+        "newmark_beta":                       0.25,
+        "newmark_gamma":                      0.5,
+        "newmark_theta":                      0.5,
+        "rayleigh_m":                         0.0,
+        "rayleigh_k":                         0.0,
+        "strategy_type":                      "newton_raphson",
+        "convergence_criterion":              "residual_criterion",
+        "displacement_relative_tolerance":    1.0E-4,
+        "displacement_absolute_tolerance":    1.0E-4,
```

Comment:
Usually we set "displacement_absolute_tolerance":    1.0E-9,
Did you set it intentially to 1e-4?

Reply:
Put them both to 1e-9, it doesn't really matter since we use residual_criterion anyway, but it's better for consistency, nice catch!